lib/galaxy/util/renamed_temporary_file.py

```diff
@@ -40,9 +40,7 @@ def __enter__(self):
     def __exit__(self, exc_type, exc_val, exc_tb):
         if exc_type is None:
             self.tmpfile.flush()
-            result = self.tmpfile.__exit__(exc_type, exc_val, exc_tb)
+            self.tmpfile.__exit__(exc_type, exc_val, exc_tb)
             os.rename(self.tmpfile.name, self.final_path)
         else:
-            result = self.tmpfile.__exit__(exc_type, exc_val, exc_tb)
-
-        return result
+            self.tmpfile.__exit__(exc_type, exc_val, exc_tb)
```

Comment:
I think we should explicitly return False, given the docs: https://docs.python.org/3/library/stdtypes.html#contextmanager.__exit__

```suggestion
            self.tmpfile.__exit__(exc_type, exc_val, exc_tb)
        return False
```

Reply:
Rebased with this change.